src/withUrlState.ts

```diff
@@ -47,8 +47,10 @@ export const html5HistoryAdapter: HistoryAdapter = {
 }
 
 export type Parse<T> = (queryString: string) => Partial<T>
+export const defaultParseOptions =  { ignoreQueryPrefix: true }
```

Comment:
Can we make this go away? What are the defaults?

Reply:
So the problem is that it doesn't insert the "?" at the start by default :/
which is a bit shite, we could do something like defining a default parser that includes that option, that way you wouldn't see it in any of the tests... 
That or use a different query string parser but 'querystring' was last updated 5 years ago...

Reply:
Ok, I would rather not have those as exports, may in the tests do `const parse = search => qs.parse(search, { ignore })`

Reply:
agreed